src/base/_forms.css

```diff
@@ -159,6 +159,21 @@ input[type="range"] {
   padding: 0;
 }
 
+input[type="range"]::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  background: var(--primary);
+  height: 18px;
+  width: 18px;
+  border-radius: 50%;
+}
+
+input[type=range]::-moz-range-thumb {
+  background: var(--primary);
+  height: 18px;
+  width: 18px;
+  border-radius: 50%;
+}
```

Comment:
Maybe better to change this to:

```
input[type="range"]::-webkit-slider-thumb {
  -webkit-appearance: none;
}

input[type="range"]::-webkit-slider-thumb,
input[type=range]::-moz-range-thumb {
  background: var(--primary);
  height: 18px;
  width: 18px;
  border-radius: 50%;
}
```

To avoid duplicate code

Reply:
@C-Lodder That isn't working in Chrome ;-)

Reply:
Oh really? Not sure why, but fair enough then

Reply:
@C-Lodder https://stackoverflow.com/questions/16982449/why-isnt-it-possible-to-combine-vendor-specific-pseudo-elements-classes-into-on

> The selector (see the Selectors module [SELECT]) consists of everything up to (but not including) the first left curly brace ({). A selector always goes together with a {}-block. When a user agent can't parse the selector (i.e., it is not valid CSS3), it must ignore the {}-block as well.